uutils-coreutils/PKGBUILD

```
@@ -39,24 +40,32 @@ nix.path = "../nix"
 END
 
   cargo update -p nix
-  cargo update -p signal-hook-mio --precise 0.2.5
-  cargo update -p mio@1.0.4 --precise 1.1.0
   cargo update -p libc --precise 0.2.178
   cargo fetch --locked --target "${RUST_CHOST}"
 }
 
 # spliting build() sometimes cause building twice at make build
 package() {
-  cd "${_realname}-${pkgver}"
+  cd "$pkgname"
+  bsdtar -a -cf docs/tldr.zip --files-from /dev/null # Mask many warnings
```

Why not properly fetch docs using `source` and `noextract` instead of faking it?
```suggestion
  cp $srcdir/tldr.zip docs/tldr.zip
```